tests/utils/pod.go

```diff
@@ -105,11 +106,12 @@ func FindPodByPrefix(clientSet *kubernetes.Clientset, namespace, prefix, labelSe
 			if foundPod {
 				return true, nil
 			}
-			return false, fmt.Errorf("Unable to find pod starting with prefix %s", prefix)
-		} else {
-			return false, err
 		}
+		return false, nil
```

Comment:
L100 above: it might help with debugging to include the ns in the Fprintfs above.

Reply:
Yes good plan, I will add that.